pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1722,6 +1723,7 @@
 
     @FieldContext(
             category = CATEGORY_LOAD_BALANCER,
+            dynamic = true,
             doc = "load balance load shedding strategy"
     )
     private String loadBalancerLoadSheddingStrategy = "org.apache.pulsar.broker.loadbalance.impl.OverloadShedder";
```

Comment:
It seems this `loadBalancerLoadSheddingStrategy` is used on broker start, dynamic won't take effect.
Please check if all these configs are only used on broker start up.

Reply:
we would like to change config without deploying new code and dynamic config helps in that case. for a safe deployment we would like keep existing config, change config dynamically and do process restart without redeploying new code.  so, this is an intentional code change.

Reply:
I get your point. But is a little confusing, if we just mark it dynamic, but we need to restart to make it take effect.

Most importantly, we need to tell the difference that which of these dynamic configs need broker restart, and which ones don't.

Reply:
yes, we can add into doc. I will add it.

Reply:
Only loadBalancerLoadSheddingStrategy needs restart? Have you checked others?